packages/harmony/demo/gen_b_content.py

```
@@ -0,0 +1,483 @@
+exec(open('gen_b_pages.py').read())
+
+pages = {}
+
+# ═════════ 1. Org dashboard (console home) ═════════
+def app_row(name, host, status_color, commit, time, repo):
+    return f'''<a class="group flex items-center gap-3 px-3 py-2.5 hover:bg-muted/40">
+      <div class="relative"><div class="size-8 rounded-lg bg-gradient-to-br from-zinc-500/40 to-zinc-700/40 border border-border grid place-items-center text-xs font-semibold">{name[0].upper()}</div>
+        <span class="absolute -bottom-0.5 -right-0.5 size-2 rounded-full border-2 border-card" style="background:hsl(var(--{status_color}))"></span></div>
+      <div class="w-52 min-w-0 flex-shrink-0"><div class="text-sm font-medium truncate">{name}</div><div class="text-xs text-muted-foreground truncate">{host}</div></div>
+      <div class="hidden min-w-0 flex-1 md:block"><div class="text-xs text-muted-foreground truncate">{commit}</div><div class="text-[11px] text-muted-foreground/70 mt-0.5">{time}</div></div>
+      <div class="hidden max-w-44 flex-shrink lg:block"><span class="raised flex items-center gap-1.5 rounded-full border border-border bg-card px-2.5 py-1 text-xs text-muted-foreground">{icon('github', 'size-3')}<span class="truncate">{repo}</span></span></div>
```

Ruff (0.16.1)

[error] Undefined name `icon`

(F821)